DESIGN.md

```diff
@@ -0,0 +1,58 @@
+# Automatisch Design Guidelines & Resources
+
+A collection of design contribution guidelines and resources for Automatisch.
+
+> **All participating designers are highly encouraged to shape and evolve these guidelines!**
+
+## 👋 Welcome
+
+Automatisch the Open Source Zapier alternative – a business automation tool that lets you connect different services like Twitter, Slack, and more to automate your business processes.
+
+You can learn more about the product on our [website](https://automatisch.io) or check out our [documentation](https://automatisch.io/docs)
+
+## 🚢 How to contribute design
+
+1. Check out open [issues](https://github.com/automatisch/automatisch/issues) here on GitHub (we label them with `design: 💅required`)
+2. Feel free to open an issue on your own if you find something you would like to contribute to the project and use the `design: 💡idea` label for it.
+3. Clone the public Figma files if available or create new ones and share them publicly
+4. Add your contributions to an issue and we promise we will review your contribution carefully and foster discussions
+
+**We encourage you to:**
+
+- Get in touch with the team by starting a discussion on [GitHub](https://github.com/automatisch/automatisch/discussions) or on our [Discord Server](https://discord.gg/dJSah9CVrC).
+- Check out our [Code of Conduct](https://github.com/myorg/.github/blob/main/CODE_OF_CONDUCT.md)
+
+## 🎭 Target audience
+
+Since we are creating a product for everyone from single persons to businesses our target audience varies heavily but our users typically have a deep desire to automate processes and are therefore a little bit techy. (e.g. Play music in my Kitchen when I receive an E-Mail)
+
+## 💅 Design relevant materials
+
+Here is a list of design relevant information and materials:
+
+### Component library
+
+We rely on the custom themed [Material UI](https://mui.com/material-ui/) component library for our Frontend App.
+
+You can find more information about [the theme](https://github.com/automatisch/automatisch/blob/main/packages/web/src/styles/theme.ts) here. 
+
+There is a community edition Figma file (beware it's not themed with our customizations!):
```

Comment:
@fivenp we purchased an MUI Figma file that's customized with our theme. However, I'm not sure what approach we should follow here. Do you have any best/standard practices on this?

We are especially concerned that the license of the Figma file differs from the community edition one. So, we don't feel much comfortable sharing it.

https://mui.com/store/items/figma-react/

Reply:
That is interesting... 

The limitation is only for editors – that means marking that resource **viewable** (and linking to it) is totally ok.

What projects typically do is: 

* Only the specified amount of editors has write/edit access to the library
* They create their mocks in a Figma file using the components from the bought library
* They do share their mocks with others but do not share the core library

I'll reach out to the creators of it and we'll see where we can take it from here :)